src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -492,7 +492,7 @@ public static class Parser
         private static readonly Regex FileExtensionRegex = new Regex(@"\.[a-z0-9]{2,4}$",
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
-        private static readonly RegexReplace SimpleTitleRegex = new RegexReplace(@"(?:(480|540|576|720|1080|2160)[ip]|[xh][\W_]?26[45]|DD\W?5\W1|[<>?*]|848x480|1280x720|1920x1080|3840x2160|4096x2160|(8|10)b(it)?|10-bit)\s*?",
+        private static readonly RegexReplace SimpleTitleRegex = new RegexReplace(@"(?:(480|540|576|720|1080|2160)[ip]|[xh][\W_]?26[45]|DD\W?5\W1|[<>?*]|848x480|1280x720|1920x1080|3840x2160|4096x2160|(8|10)(b(?![a-z0-9])|bit)|10-bit)\s*?",
```

Comment:
```suggestion
        private static readonly RegexReplace SimpleTitleRegex = new RegexReplace(@"(?:(480|540|576|720|1080|2160)[ip]|[xh][\W_]?26[45]|DD\W?5\W1|[<>?*]|848x480|1280x720|1920x1080|3840x2160|4096x2160|(?<![]a-f0-9])(8|10)b(it)?|10-bit)\s*?",
```

This works with the example title provided.

Reply:
I combined our two patterns. Your look behind, but with my look ahead.

Some mocked up example parses:
```
[DHD] Series Title! - 01 - Episode Title [1080p][10bit][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p 10bit][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p 10b][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p][10b][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p 10b][8B15AF29]
[DHD] Series Title! - 01 - Episode Title [1080p][10b][8B15AF29]
[DHD] Series Title! - 01 - Episode Title [1080p][8bit][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p 8bit][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p 8b][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p][8b][15AF298B]
[DHD] Series Title! - 01 - Episode Title [1080p 8b][8B15AF29]
[DHD] Series Title! - 01 - Episode Title [1080p][8b][8B15AF29]
```

With just the suggested change:
![image](https://github.com/Sonarr/Sonarr/assets/21229471/10535d49-7a23-48f7-b1b1-521e033bd854)

With the one I committed:
![image](https://github.com/Sonarr/Sonarr/assets/21229471/178fc38a-2094-43fb-99c4-5eb00ca90f95)

Reply:
Currently only tests for `8b` and `10b` are covered, would be good to add additional ones.

Reply:
What additional ones where you thinking of?
I split off the explicit 8b and 10b within the hashes to another function, with begins with, ends with, and contains variants.

Reply:
Perfect, I think I just missed them as I reviewed.